### PR TITLE
CI: Add codeowners

### PR DESCRIPTION
**Pull request by SaschaMann**
(mostly because I'm too lazy to request your review manually but this probably makes sense given the earlier discussion)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# CI
+.github/**/*		@iHiD @ErikSchierboom
```
Comment (iHiD) [inline .github/CODEOWNERS:2]: This is the team I've made based on our discussions earlier.

```suggestion
.github/**/*		@exercism/maintainers-admin
```

**Comment by iHiD** [comment]
(cc @ErikSchierboom)

**Comment by iHiD** [comment]
Thanks @SaschaMann 🙂

**Comment by SaschaMann** [comment]
Thanks for supporting my laziness :P

**Comment by ErikSchierboom** [comment]
Thanks!

**Pull request merged**